include/caffe/net.hpp

```diff
@@ -25,6 +25,7 @@ class Net {
  public:
   explicit Net(const NetParameter& param, const Net* root_net = NULL);
   explicit Net(const string& param_file, Phase phase,
+      const int level = 0, const vector<string>* stages = NULL,
```

Comment:
Does anyone use the `root_net` parameter? I'd like to keep the `phase`, `level` and `stages` parameters next to each other since they're naturally grouped. If anything, I'd prefer to move `root_net` before `phase`, but that would definitely break existing code.